packages/create-vite/solid-js/vite.config.js

```diff
@@ -0,0 +1,12 @@
+import { defineConfig } from 'vite'
+import solidPlugin from 'vite-plugin-solid'
+
+export default defineConfig({
+  plugins: [solidPlugin()],
+  server: {
+    port: 3000,
+  },
+  build: {
+    target: 'esnext',
+  },
```

Comment:
We should probably remove these as well to match the other config.

Reply:
I thought about removing it, but I wasn't sure so I waited for the review. Thanks for notifying me.